src/Board.spec.ts

```diff
@@ -1,17 +1,34 @@
 import expect from "expect";
-import { getInitialBoard } from './Board'
+import { getInitialBoard, getBoardFromFile, INITIAL_BOARD } from "./Board";
+import { close } from './UserInput';
 
 describe("Board test", () => {
+  afterAll(() => {
+    close();
+  });
+
   it("should turn over the initial game board", () => {
     const firstBoard = getInitialBoard();
-    expect(firstBoard).toStrictEqual([
-        [1, 1, 0, 0, 0, 2, 2], 
-        [1, 1, 0, 3, 0, 2, 2], 
-        [0, 0, 3, 3, 3, 0, 0], 
-        [0, 3, 3, 3, 3, 3, 0], 
-        [0, 0, 3, 3, 3, 0, 0], 
-        [2, 2, 0, 3, 0, 1, 1], 
-        [2, 2, 0, 0, 0, 1, 1]
-    ]);
+
+    expect(firstBoard).toStrictEqual(INITIAL_BOARD);
   });
+
+  it("should turn over a custom game board with good path", async () => {
+    const customGameBoard = [
+      [1, 2],
+      [0, 3],
+    ];
+
+    const board = await getBoardFromFile("src/board.json");
```

Comment:
To get board in src folder is not a good idea imo. Think about how your PO will test it

Reply:
I move the file on an assets folder. 
The PO can use an absolute path, so do I need to change anything?